### PR TITLE
Add circleci

### PR DESCRIPTION
**Pull request by awm33**
### Motivation

We would like to automate continuous integration testing.

Closes
https://github.com/cognoma/core-service/issues/9

### Implementation Notes

I actually found an issue with my migration dependencies on django-genes that probably would have caused issues on machines other than my own, since I ran `makemigrations` on that app individually.

@dhimmel This is a good example for what I would consider very squahable. The commits in this PR are small and not very meaningful on their own.

### Functional Tests

Having CircleCI automatically pick up my changes.

### CODE REVIEW

**Comment by awm33** [comment]
@dhimmel @cgreene Here we go! You should see the ❌ s and ✅ s from CircleCI next to the commits

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+machine:
+  python:
+    version: 3.5.1
+  services:
+    - postgresql
+
+database:
+  override:
+    - psql -c "CREATE ROLE app PASSWORD 'password' SUPERUSER CREATEDB INHERIT LOGIN"
```
Comment (dhimmel) [inline circle.yml]: Adding a newline here is probably good.

**Comment by awm33** [comment]
From the build (click on the ❌  or ✅ ) [https://circleci.com/gh/cognoma/core-service/6](https://circleci.com/gh/cognoma/core-service/6)

> Ran 26 tests in 0.405s
 
@dhimmel It's running tests

File: circle.yml (hunk 1 above)
Comment (awm33) [inline circle.yml]: Added

**Comment by dhimmel** [comment]
I see https://circleci.com/docs/language-python/:

> CircleCI automatically runs `tox` when we find a `tox.ini` file, and runs `nosetests` when we find a `unittest.py` file. If you are using Django, then Circle will run `manage.py test`.

So CircleCI picks up that Django is installed from `requirements.txt` and `manage.py`.

**Comment by awm33** [comment]
@dhimmel

**Pull request merged**